imports/ui/components/externalApplications/helpModal.html

```diff
@@ -0,0 +1,48 @@
+<template name="AppHelpModal">
```

Comment:
need something that links it to external apps. i.e. ManageExternalAppsHelpModal

Reply:
updated